servicetalk-grpc-protoc/src/main/java/io/servicetalk/grpc/protoc/Generator.java

```diff
@@ -116,12 +122,14 @@
 import static io.servicetalk.grpc.protoc.Words.onClose;
 import static io.servicetalk.grpc.protoc.Words.request;
 import static io.servicetalk.grpc.protoc.Words.requestEncoding;
+import static io.servicetalk.grpc.protoc.Words.responseWriter;
 import static io.servicetalk.grpc.protoc.Words.routes;
 import static io.servicetalk.grpc.protoc.Words.rpc;
 import static io.servicetalk.grpc.protoc.Words.service;
 import static io.servicetalk.grpc.protoc.Words.strategy;
 import static io.servicetalk.grpc.protoc.Words.strategyFactory;
 import static io.servicetalk.grpc.protoc.Words.supportedMessageCodings;
+import static java.lang.System.lineSeparator;
```

Comment:
I wonder if we should just be using "\n". It is unlikely that we will be generating javadoc on Windows but there it would be "\r\n".

Reply:
although we currently don't run CI on windows we should still maintain compatibility. we should enable windows CI in the future (after existing flaky tests are addressed).